uncrustify-riot.cfg

```diff
@@ -24,6 +24,8 @@ nl_func_var_def_blk    = 1        #
 nl_fcall_brace         = remove   # "list_for_each() {" vs "list_for_each()\n{"
 nl_fdef_brace          = add      # "int foo() {" vs "int foo()\n{"
 nl_collapse_empty_body = true     # set while(){\n} to while(){}
+nl_end_of_file         = add      # fix no newline at end of file
```

Comment:
just a minor minor minor thing :). Can you align the # comments here?

Reply:
oops, just hit "comment".

Reply:
I don't understand, they are already aligned...